spark/src/main/java/org/apache/iceberg/actions/ExpireSnapshotsActionResult.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.actions;
+
+public class ExpireSnapshotsActionResult {
+
+  private final Long dataFilesDeleted;
+  private final Long manifestFilesDeleted;
+  private final Long manifestListsDeleted;
+
+  public ExpireSnapshotsActionResult(Long dataFilesDeleted, Long manifestFilesDeleted, Long manifestListsDeleted) {
+    this.dataFilesDeleted = dataFilesDeleted;
+    this.manifestFilesDeleted = manifestFilesDeleted;
+    this.manifestListsDeleted = manifestListsDeleted;
+  }
+
+  public Long getDataFilesDeleted() {
```

Comment:
Minor: We don't use `get` because it doesn't generally add any value. In most cases, the right thing to do is to use a more descriptive verb, like `load` or `create`. If there isn't a more descriptive verb because it is just accessing a member field, then we omit `get`. That's more readable and the convention in most other JVM languages.

Reply:
sgtm